src/sidebar/templates/group-list.html

```diff
@@ -1,22 +1,32 @@
 <div class="pull-right"
      dropdown
      keyboard-nav>
+  <!-- Show a drop down menu if showGroupsMenu is true. -->
   <div class="dropdown-toggle"
         dropdown-toggle
-        data-toggle="dropdown"
-        role="button"
-        title="Change the selected group">
+        data-toggle="dropdown" ng-if="vm.showGroupsMenu()">
     <img class="group-list-label__icon group-list-label__icon--organization"
          ng-src="{{ vm.focusedIcon() }}"
          alt="{{ vm.orgName(vm.groups.focused().id)}}"
          ng-if="vm.focusedIcon()">
          <i class="group-list-label__icon h-icon-{{ vm.focusedIconClass() }}"
              ng-if="!vm.focusedIcon()"></i><!-- nospace
-    !--><span class="group-list-label__label">{{vm.groups.focused().name}}</span><!-- nospace
-      !--><i class="h-icon-arrow-drop-down"></i>
+    !--><h2 class="group-list-label__label has-toggle"><a class="group-list-label__toggle">{{vm.groups.focused().name}}</a><!-- nospace
+      !--><i class="h-icon-arrow-drop-down"></i></h2>
   </div>
-  <div class="dropdown-menu__top-arrow"></div>
-  <ul class="dropdown-menu pull-none" role="menu">
+  <!-- Do not show a drop down menu if showGroupsMenu is false. -->
+  <div ng-if="!vm.showGroupsMenu()">
+    <img class="group-list-label__icon group-list-label__icon--organization"
+         ng-src="{{ vm.focusedIcon() }}"
+         alt="{{ vm.orgName(vm.groups.focused().id)}}"
+         ng-if="vm.focusedIcon()">
+         <i class="group-list-label__icon h-icon-{{ vm.focusedIconClass() }}"
+             ng-if="!vm.focusedIcon()"></i><!-- nospace
+    !--><h2 class="group-list-label__label">{{vm.groups.focused().name}}</h2>
```

Comment:
It is a bit unfortunate that these several lines of markup had to be duplicated. The general pattern I would like us to use when there is significant duplication is to extract the duplicated markup into a separate component, just like extracting a function in regular code.

Unfortunately in Angular there is quite a bit of boilerplate involved in creating a new component compared to eg. React, so that's probably best left for a separate PR.

Reply:
I'm currently out of my depth RE: creating a new Angular component, but agree that refactoring this is probably best separated out into another PR.

Reply:
I felt like there would be a lot of duplication between the two controllers so I chose to add it to the existing one rather than make two.